src/components/Cards/calculator.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <div class=" lg:w-2/5 w-full  bg-white lg:flex-col pb-16  rounded lg:my-20 my-0 lg:mx-20 mx-0 h-full  hover:shadow-2xl">
+          <h3 class=" tracking-wider mb-32 leading-6 text-2xl font-bold text-center px-8 mt-16 text-black  ">
+                   Your initial payment is <br><p class="underline pt-8"> ₦20,900.00</p>
```

Comment:
why do we have hardcoded figures?

Reply:
I just used it ti design the UI, the data will be fetched... I am not done with it